CONTRIBUTING.md

```diff
@@ -46,3 +46,13 @@ The outline above is an extremely simplified version. In real life you have to d
 - Make sure test-suite passes: `npm test`
 - Squash your commits
 - Write a convincing description of your PR and why we should land it
+
+## Release Checklist
+
+- Run [`npm version patch`](https://docs.npmjs.com/cli/version) to update the version number. Use `minor` or `major` instead of `patch` if needed (see [semver.org](http://semver.org/) for details).
+- Open a [pull request](https://github.com/OctoLinker/browser-extension/pulls) with the new version.
+- Once the pull request is merged in, tag the resulting commit as `vX.Y.Z` (where `X`, `Y`, `Z` are the major, minor, and patch versions).
```

Comment:
Yes, but since we're using GitHub's "squash and merge" feature, the tag will not reference the correct commit hash, as it changes during the squash/merge.